code/__DEFINES/subsystems.dm

```diff
@@ -124,7 +125,7 @@
 #define RUNLEVELS_DEFAULT (RUNLEVEL_SETUP | RUNLEVEL_GAME | RUNLEVEL_POSTGAME)
 
 
-
+// Truly disgusting, TG. Truly disgusting.
```

Comment:
this is frankly a beautiful macro, and you should shut up

Reply:
uh oh

Reply:
it's art, you wouldn't UNDERSTAND

Reply:
>art

more like stinky